Sources/Valkey/RESP/RESPStringRenderable.swift

```diff
@@ -20,7 +20,7 @@ import FoundationEssentials
 import Foundation
 #endif
 
-/// A type that can be rendered as a single bulk string.
+/// A string-like type that the command encoder can render to send as a Valkey command.
```

Comment:
This isn't totally correct. RESPStringRenderable is a type that can be rendered as a RESP bulkString. It would never be used for a complete command
```suggestion
/// A type that the command encoder can render as a single string in a Valkey command.
```